### PR TITLE
Update MixUp Example to Keras 3

### PR DESCRIPTION
**Pull request by nkovela1**

### CODE REVIEW

File: examples/vision/md/mixup.md [hunk 1]
```diff
@@ -210,16 +222,23 @@ network.
 
 ```python
 initial_model = get_training_model()
-initial_model.save_weights("initial_weights.h5")
+initial_model.save_weights("initial_weights.weights.h5")
 ```
 
+<div class="k-default-codeblock">
+```
+/opt/conda/envs/keras-tensorflow/lib/python3.10/site-packages/keras/src/layers/convolutional/base_conv.py:99: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
```
Comment (fchollet) [inline examples/vision/md/mixup.md:230]: Please fix.

File: examples/vision/mixup.py [hunk 2]
```diff
@@ -30,18 +30,27 @@
...
-This example requires TensorFlow 2.4 or higher.
 """
 
 """
 ## Setup
 """
 
+import os
+
+os.environ["KERAS_BACKEND"] = "tensorflow"
```
Comment (fchollet) [inline examples/vision/mixup.py:41]: Is the example TF-specific or can it run with another backend?

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: examples/vision/mixup.py (hunk 2 above)
Comment (nkovela1) [reply examples/vision/mixup.py:41]: Yes, this example is TF-only

File: examples/vision/md/mixup.md (hunk 1 above)
Comment (nkovela1) [reply examples/vision/md/mixup.md]: Fixed, thanks!

**Review by fchollet — APPROVED** [review]
LGTM, thank you

**Pull request merged**